data/json/items/migration.json

```diff
@@ -1823,5 +1823,45 @@
     "type": "MIGRATION",
     "replace": "folded_inflatable_boat",
     "reset_item_vars": true
+  },
+  {
+    "id": "5x50dart",
```

Comment:
OK so if this migration exists here then we need to rename the id's on anything in Aftershock that has the same id as something being migrated.   I'd add an `afs_` prefix to the front of the id.